### PR TITLE
Added a new option to set the (optional) max length of inactive tab titles

### PR DESCRIPTION
**Pull request by kirameister**

### CODE REVIEW

File: lua/tabline.lua [hunk 1]
```diff
@@ -42,11 +43,18 @@ local function tabline(options)
...
         -- buf name
         s = s .. options.brackets[1]
+        local pre_title_s_len = string.len(s)
         if bufname ~= '' then
             s = s .. icon .. fn.fnamemodify(bufname, ':t')
         else
             s = s .. options.no_name
         end
+        if options.inactive_tab_max_length
+            and options.inactive_tab_max_length ~= 0
```
Comment (kirameister) [inline lua/tabline.lua]: Actually, this condition could be set to something like `options.inactive_tab_max_length > 0`. Please let me know if that's what you prefer (to be on a safe side).

Comment (crispgm) [reply lua/tabline.lua]: > Actually, this condition could be set to something like `options.inactive_tab_max_length > 0`. Please let me know if that's what you prefer (to be on a safe side).

This sounds good to me.

**Comment by crispgm** [comment]
I like the idea. But should we set default to `0` instead of `10`?

File: lua/tabline.lua (hunk 1 above)
Comment (kirameister) [reply lua/tabline.lua]: Thanks. Update coming up..

**Comment by kirameister** [comment]
> I like the idea. But should we set default to `0` instead of `10`?

Thank you. I've just pushed the update as suggested.

**Comment by crispgm** [comment]
LGTM. Thank you.

**Pull request merged**